packages/wrangler/src/dev.tsx

```diff
@@ -277,6 +277,10 @@ export function devOptions(yargs: CommonYargsArgv) {
 				// Yargs requires this to type log-level properly
 				default: "log" as LoggerLevel,
 			})
+			.option("show-interactive-dev-session", {
+				describe: "Show interactive dev session",
```

Comment:
It might be worth putting the default into the `describe` since we don't want to tell yargs about it directly...

```suggestion
				describe: "Show interactive dev session (defaults to true)",
```

Reply:
Yeah, makes sense. Have added in [4d374f9](https://github.com/cloudflare/workers-sdk/pull/5099/commits/4d374f91b220510df6fe77c1900ea446879a5cb8)